src/main/java/org/sasanlabs/service/vulnerability/pathTraversal/PathTraversalVulnerability.java

```diff
@@ -90,8 +90,7 @@
             value = LevelConstants.LEVEL_1,
             descriptionLabel = "PATH_TRAVERSAL_URL_CONTAINING_FILENAME",
             htmlTemplate = "LEVEL_1/PathTraversal",
-            parameterName = URL_PARAM_KEY,
-            sampleValues = {SAMPLE_VALUE_FILE_NAME})
```

Comment:
can you please remove this constant too if not used anywhere else.

Reply:
are you referring to the parameterName constant? If so, it looks like it's used in the same exact places that sampleValues was used. once you confirm, i'll remove that constant from the VulnerableAppRequestMapping interface and all of its pointers.

Reply:
No no, I was talking about string constant "SAMPLE_VALUE"

Reply:
It looks like this constant is used by ALLOWED_FILE_NAMES: 
https://github.com/SasanLabs/VulnerableApp/blob/e4ee8917cbd43c9eeae04ad75db7d76290753327/src/main/java/org/sasanlabs/service/vulnerability/pathTraversal/PathTraversalVulnerability.java#L43-L44
and this list is used multiple times by this.readfile:
https://github.com/SasanLabs/VulnerableApp/blob/e4ee8917cbd43c9eeae04ad75db7d76290753327/src/main/java/org/sasanlabs/service/vulnerability/pathTraversal/PathTraversalVulnerability.java#L219-L226

So I guess since it's being used there, we shouldn't remove that unless you also want to remove ALLOWED_FILE_NAMES.

Reply:
oh ok got it, then can you please remove this constant and directly add the value of the constant directly like `ALLOWED_FILE_NAMES = 
         Arrays.asList("UserInfo.json", "OwaspAppInfo.json"); `